### ISSUE TITLE
Demo?

### ISSUE THREAD

**Issue by intrepidOlivia** [sinsau/ng2-json-view#3]
There's a few different JSON viewers out there. Do you have a demo available to easily see if this is the solution that I need?

Thanks,

intrepidOlivia

**Comment by sinsau** [comment]
I will try to add demo, that will be good, bit busy these days but will try to work it out.

**Comment by intrepidOlivia** [comment]
Of course. And it's your project, it's up to you how you want to present it! Sorta depends on how much you want strangers using your code, I think. :)
